crowbar_framework/app/models/neutron_service.rb

```diff
@@ -277,6 +299,9 @@ def apply_role_pre_chef_call(old_role, role, all_nodes)
 
     network_nodes.each do |n|
       net_svc.allocate_ip "default", "public", "host",n
```

Comment:
> net_svc.allocate_ip "default", "public", "host",n
> 
> can be removed,

That's not really true. It could be changed to an appropriate enable_interface call. But let's worry about that in the other pull request (#227).